backend/src/test/java/de/dontletyoudie/backend/cucumberglue/CucumberMySteps.java

```diff
@@ -0,0 +1,45 @@
+package de.dontletyoudie.backend.cucumberglue;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.cucumber.spring.CucumberContextConfiguration;
+import org.junit.jupiter.api.Assertions;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@CucumberContextConfiguration
+public class CucumberMySteps {
+
+    @LocalServerPort
+    String port;
+
+    ResponseEntity<String> lastResponse;
+
+    @When("the client calls endpoint {string}")
+    public void whenClientCalls(String url) {
+        lastResponse = new RestTemplate().exchange("http://localhost:" + port + url,
+                HttpMethod.GET,
+                null,
+                String.class);
+    }
+
+
+    @Then("response status code is {int}")
+    public void thenStatusCode(int expected) {
+        assertThat("status code is " + expected,
+        lastResponse.getStatusCodeValue() == expected);
```

Comment:
```suggestion
        Assertions.assertThat(lastResponse.getStatusCodeValue())
                .isEqualTo(expected);
```

just a suggestion for using assertj